src/main/java/frc/robot/commands/ToggleDriveTarget.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+public class ToggleDriveTarget extends CommandBase{
```

Comment:
It feels like we want these on separate buttons so having this command be SetDriveRotationTarget and accepting the rotation target as a constructor argument. would make sense. 

The driver should have plenty of buttons so this would be ok, maybe Y and A.

The thought is driver the is translating and rotating around the field as they see fit, they press Y or A and it always goes to a consistent orientation but they can cancel at any time by moving the rotation joystick